File: program/print-reverse-of-a-string-by-recursion/reverse number.py
def recursum(number,reverse):
  if number==0:
    return reverse
  remainder = int(number%10)
  reverse = (reverse*10)+remainder
  return recursum(int(number/10),reverse)

num = int(input("Enter the Number:"))
reverse = 0
print(recursum(num,reverse))

File: program/reverse-of-a-number/ReverseNumber.c
#include <stdio.h>

void main(){
    int num,r,sum=0,t;

    printf("Input a number: ");
    scanf("%d",&num);

    for(t=num;num!=0;num=num/10){
         r=num % 10;
         sum=sum*10+r;
    }
printf("The number in reverse order is : %d \n",sum);
}
